lib/parser/demunge.ts

```diff
@@ -11,10 +12,10 @@ export default demunge;
 /**
  * Demunges the given policy object.
  * @param policy  The policy object to demunge
- * @param apiRoot The root URL of the Snyk API
+ * @param apiRoot A string or func calculating the base URL for the Snyk API
  * @returns The demunged policy object
  */
-function demunge(policy: Policy, apiRoot = '') {
+function demunge(policy: Policy, apiRoot?: ApiRootFunction | string) {
```

Comment:
Are we intentionally changing the default? I think this may now throw when `undefined` is passed. Maybe add another quick test to make sure that it's behaving as expected

Reply:
ah, yes, you are right! Downgrade to a "nit" as I still think it's good to make the default explicit on the function signature when possible